packages/tupaia-web/src/layout/ProjectCardList/ProjectCard.tsx

```diff
@@ -162,11 +165,24 @@ export const ProjectPendingLink = () => (
   </OutlineLink>
 );
 
-export const ProjectAllowedLink = ({ url, isLandingPage }: LinkProps) => (
-  <BaseLink to={url} target={isLandingPage ? '_blank' : '_self'}>
-    View project
-  </BaseLink>
-);
+type ProjectAllowedLinkProps = LinkProps & {
+  projectId: string;
+};
+
+export const ProjectAllowedLink = ({ projectId, url, isLandingPage }: ProjectAllowedLinkProps) => {
+  const navigate = useNavigate();
+  const { mutate } = useEditUser(() => {
+    if (isLandingPage) {
+      window.open(url, '_blank');
```

Comment:
Personally I would prefer if the button was still a link if linking to another page, instead of programatically navigating, just because it breaks some accessibility rules